.travis.yml

```diff
@@ -1,13 +1,22 @@
 language: go
+
 os:
   - windows
   - linux
   - osx
+
+branches:
+  only:
+    - master
+
 go:
-  - "1.11.x"
-  - "1.12.x"
+  - "1.11.13"
+  - "1.12.9"
```

Comment:
why make them explicit, and not use `.x`? this will force someone to manually update them every few weeks. you can generally rely on bugfix releases not breaking anything.

Reply:
Simply because it's then not a lottery with `gimme`. But I'm not fixed on this way or the other...

Reply:
Unless gimme or the `.x` bits give you trouble, I'd just leave it as is. I haven't had any issues for years.

One case I do pin the version is when deploying or building release binaries, or when testing at work, since we want to carefully use one Go version everywhere. But for an open source project, I think using the latest bugfix is much easier.

Reply:
Sounds like a good practical approach :)